### PR TITLE
WLA-554: Prevent Decision/Direction Issued notifications going out f…

### PR DESCRIPTION
**Pull request by ruban72**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -288,6 +288,18 @@ notification:
...
+    question_round_issued:
+      emailId: ccfbfcac-847d-4e55-955c-096802c131a0
+    follow_up_question_round_issued:
+      emailId: 71efa426-bfa9-473e-b3cf-da5d99e39461
+    question_deadline_elapsed:
+      emailId: 48f2b7d1-cb71-4f60-b145-dfcf63cc4fcd
+    question_deadline_reminder:
+      emailId: 29820a6a-1aa4-4518-8d08-f06f09c5d808
+    continuous_online_hearing_relisted:
+      emailId: 46cc347a-1a5c-40ce-993a-49769dfce626
```
Comment (jackmaloney) [inline src/main/resources/application.yaml:300]: How have these ended back in? Look like they are related to COR functionality which has been removed.

File: src/main/resources/application.yaml [hunk 2]
```diff
@@ -318,6 +330,8 @@ notification:
         docmosisId: TB-SCS-GNO-ENG-00510.docx
       representative:
         docmosisId: TB-SCS-GNO-ENG-00512.docx
+    corDecision:
```
Comment (jackmaloney) [inline src/main/resources/application.yaml:333]: Same

File: src/main/resources/application.yaml [hunk 3]
```diff
@@ -604,6 +618,18 @@ notification:
...
+    follow_up_question_round_issued:
+      emailId: 71efa426-bfa9-473e-b3cf-da5d99e39461
+    question_deadline_elapsed:
+      emailId: 48f2b7d1-cb71-4f60-b145-dfcf63cc4fcd
+    question_deadline_reminder:
+      emailId: 29820a6a-1aa4-4518-8d08-f06f09c5d808
+    continuous_online_hearing_relisted:
+      emailId: 46cc347a-1a5c-40ce-993a-49769dfce626
+    decision_issued:
+      emailId: 62e72e11-5eb6-4048-83f4-0b0c5c59fcb9
```
Comment (jackmaloney) [inline src/main/resources/application.yaml:632]: Same

File: src/main/resources/application.yaml [hunk 4]
```diff
@@ -620,13 +646,29 @@ notification:
...
+      representative:
+        docmosisId: TB-SCS-GNO-WEL-00475.docx
     issueFinalDecision:
       appellant:
         docmosisId: TB-SCS-GNO-ENG-00454.docx
       appointee:
         docmosisId: TB-SCS-GNO-ENG-00454.docx
       representative:
         docmosisId: TB-SCS-GNO-ENG-00455.docx
+    corDecision:
```
Comment (jackmaloney) [inline src/main/resources/application.yaml:670]: Same

File: src/main/resources/application.yaml (hunk 1 above)
Comment (ruban72) [reply src/main/resources/application.yaml]: Have removed these entries

File: src/main/resources/application.yaml (hunk 3 above)
Comment (ruban72) [reply src/main/resources/application.yaml]: Have removed these entries

File: src/main/resources/application.yaml (hunk 4 above)
Comment (ruban72) [reply src/main/resources/application.yaml]: Have removed these entries

**Review by jackmaloney — APPROVED** [review]

**Pull request merged**